timesketch/lib/testlib.py

```diff
@@ -90,7 +90,7 @@ def __init__(self):
         self.indices = MockOpenSearchIndices()
 
     def search(
-        self, index, body, size=0, search_type=None
+        self, index, body, size=0, search_type=None, _source_include=None
```

Comment:
needed to add this to support the tag events unit tests that where previously not there